standard/documentation-comments.md

```diff
@@ -61,6 +61,16 @@ Although developers are free to create their own set of tags, a recommended set
 
 Note carefully that the documentation file does not provide full information about the type and members (for example, it does not contain any type information). To get such information about a type or member, the documentation file must be used in conjunction with reflection on the type or member.
 
+A partial type or a partial non-type member (such as a method) can be declared in multiple parts, each of which can be in one or more compilation units, and each of which can have one or more documentation comments. A partial non-type member typically has a “defining partial declaration” and an “implementing partial declaration.”
+
+For a partial type, the document comments that apply directly to that type, if any, from each of its parts, are all written to the documentation file in some unspecified order.
+
+For a partial non-type member
+
+- If a defining partial declaration has no corresponding implementing partial declaration, any documentation comments in that defining partial declaration are ignored (as that declaration will be removed).
+- Otherwise, if the implementing partial declaration has any documentation comments, they are written to the documentation file in some unspecified order, and any documentation comments in the defining partial declaration are ignored.
```

Comment:
Both here and in the comment below, we can remove "in some unspecified order". There's only *one* implementing partial declaration, and only *one* defining partial declaration. (Compare that with partial type declarations, where there really are multiple potential comment sources, so we need the "in some unspecified order" bit.)

Reply:
This might be the right time to adopt the terms used in the C# 13 docs for new style partial members.  For all partial members (including methods) must have an implementation, we've used the terms *declaring declaration* and *implementing declaration*. Adopting these terms made it much easier to describe these members where there must be exactly one implementation and exactly one declaration.

Reply:
Consider a partial method having a summary tag and a params tag. Should we require they be written out in the same order as they appear in the source? I presumed "No", hence the use of "unspecified order."

Reply:
But why highlight that here? Do we say the same thing about documentation comments for *non*-partial methods? (The word "unspecified" doesn't occur in annex D.2, and "documentation" doesn't occur in classes.md.)

Basically unlike partial types, there's one one source for documentation. Partial types really *are* unusual in terms of doc comments, but partial methods aren't - so I don't think we should be specifying them as if the documentation for partial methods has less of an ordering requirement than documentation for non-partial methods.

Reply:
Resolved for @RexJaeschke  to remove "in some unspecified order" just for methods.

Reply:
OK, will remove the ordering for methods.